tests/fixers/test_email_apis.py

```diff
@@ -0,0 +1,164 @@
+from __future__ import annotations
+
+from functools import partial
+
+from django_upgrade.data import Settings
+from tests.fixers import tools
+
+settings = Settings(target_version=(6, 0))
+check_noop = partial(tools.check_noop, settings=settings)
+check_transformed = partial(tools.check_transformed, settings=settings)
+
+
+def test_unmatched_import():
```

Comment:
sort the tests to have the noop ones first

test every line of the `visit_Call` function

add at least one test per email function